SocketRocket/SRWebSocket.m

```diff
@@ -47,6 +47,7 @@
 #error SocketRocket must be compiled with ARC enabled
 #endif
 
+#import "TargetConditionals.h"
```

Comment:
Nit: Use angular brackets for this import, since it's coming from /usr/include
Aka `#import <TargetConditionals.h>`, also might be a good idea to move it next to other framework imports instead of it being below our custom defines.